low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -79,6 +80,47 @@ def dump_global_config(self):
         Conf.save(GLOBAL_CONFIG_INDEX)
 
     def validate(self):
+        """Check below requirements are met in setup.
+
+        1. required python 3rd party packages are installed
+        2. required rpm dependencies are installed
+        3. product specified in global config is supported by SSPL
+        """
+        # SSPL python 3rd party package dependencies
+        pip3_3ps_packages_main = {
+            "cryptography": "2.3",
+            "jsonschema": "3.2.0",
+            "pyinotify": "0.9.6",
+            "python-daemon": "2.2.4",
+            "requests": "2.25.1",
+            "zope.component": "4.6.2",
+            "zope.event": "4.5.0",
+            "zope.interface": "5.2.0"
+        }
+        # SSPL 3rd party RPM dependencies
+        rpm_3ps_packages = {
```

Comment:
these rpm versions also taken into consideration by RE ?

Reply:
Yes, Shailesh agreed.
https://github.com/Seagate/cortx-re/pull/275 has version mapped for requested packages.